src/components/common/ExchangeRate.js

```diff
@@ -69,9 +69,8 @@ class ExchangeRateComponent extends React.Component<Props & ThemeProps> {
         <FiatText
           format="raw"
           nativeCryptoAmount={primaryInfo.displayDenomination.multiplier}
-          cryptoCurrencyCode={primaryInfo.exchangeCurrencyCode}
-          isoFiatCurrencyCode={secondaryInfo.exchangeCurrencyCode}
-          cryptoExchangeMultiplier={primaryInfo.exchangeDenomination.multiplier}
+          tokenId={primaryInfo.exchangeCurrencyCode}
```

Comment:
You cannot pass a currencyCode as a tokenId. Currency codes are things like "BTC", while tokenId's are either a contract address or `undefined`.